rosidl_generator_java/cmake/rosidl_generator_java_generate_interfaces.cmake

```diff
@@ -30,6 +30,7 @@ if(NOT ANDROID)
   find_package(JNI REQUIRED)
 endif()
 include(UseJava)
+include(JavaExtra)
```

Comment:
> Do we need to include this? It doesn't look like the added content below relies on JavaExtra.

You are right, JavaExtra is not used in this file. It is used in custom_command.cmake. But since custom_command is ["copied" to different directories](https://github.com/gonzodepedro/ros2_java/blob/486c7447a530edb17a3363164bcf527c03358ce1/rosidl_generator_java/cmake/rosidl_generator_java_generate_interfaces.cmake#L178) I thought it was a good idea to add it here.

Reply:
I see. I guess we should consider moving the dependencies into `custom_command.cmake`, but I'm okay with it as-is for now.